src/Workleap.DomainEventPropagation.Subscription/DomainEventGridWebhookHandler.cs

```diff
@@ -77,7 +78,7 @@ private static IEnumerable<Assembly> GetAssemblies()
         // has references that would make a .Net Framework (as opposed to netstandard or core) project to fail
         var domainEventType = typeof(IDomainEvent);
         var domainEventAssemblies = AppDomain.CurrentDomain.GetAssemblies()
-            .Where(x => x.FullName.StartsWith("Workleap") && !x.FullName.StartsWith("Workleap.EventPropagation.Common,"))
+            .Where(x => x.FullName != null && x.FullName.StartsWith("Workleap") && !x.FullName.StartsWith("Workleap.EventPropagation.Common,"))
```

Comment:
We might want to create a story to change this behavior, not all assemblies are workleap in every products